examples/dashboard-dash/README.md

```diff
@@ -0,0 +1,150 @@
+                        html.Label("Dataset"),
+                        dcc.Dropdown(
+                            ["MNIST-Digits", "MNIST-Fashion"], "MNIST-Digits", id="dataset_dropdown"
+                        ),
+                    ],
+                    style={"padding": 10, "flex": 1},
+                ),
+                html.Div(
+                    children=[
+                        html.H1("Output"),
+                        dcc.Loading(
+                            id="loading-1",
+                            children=[dcc.Graph(id="graph")],
+                            type="circle",
+                        ),
+                    ],
+                    style={"padding": 10, "flex": 3},
+                ),
+            ],
+            style={"display": "flex", "flex-direction": "row"},
+        ),
+    ]
+)
+```
+
+### Define the Callbacks
+Dash uses callbacks to run functions based on user input. For this example, we define a single callback to update the graph when the dataset changes.
+
+Dash uses `Input` and `Output` to define callback inputs and outputs. The first parameter is the id of the associated component in the layout, and the second is the type of information passed to and from the function.
+
+For this example, the callback downloads the correct dataset based on the dropdown selection, runs `fit_transform` on the dataset to create the projection, and finally creates a 3D scatter plot of the result.
+
+``` python
+@app.callback(
+    Output("graph", "figure"),
+    Input("dataset_dropdown", "value"),
+)
+def update_figure(selected_dataset):
+    if selected_dataset == "MNIST-Digits":
+        X = pd.read_csv(
+            "https://saturn-public-data.s3.us-east-2.amazonaws.com/MNIST-1000/mnist-1000-input.csv"
+        )
+        y = pd.read_csv(
+            "https://saturn-public-data.s3.us-east-2.amazonaws.com/MNIST-1000/mnist-1000-labels.csv"
+        )
+        y = np.unique(y, return_inverse=True)[1]
+
+    elif selected_dataset == "MNIST-Fashion":
+        X = pd.read_csv(
+            "https://saturn-public-data.s3.us-east-2.amazonaws.com/MNIST-1000/fashion-1000-input.csv"
+        )
+        y = pd.read_csv(
+            "https://saturn-public-data.s3.us-east-2.amazonaws.com/MNIST-1000/fashion-1000-labels.csv"
+        )
+        y = np.unique(y, return_inverse=True)[1]
+
+    else:
+        return None, "Please select a dataset."
+
+    umap_3d = UMAP(n_components=3, init="random", random_state=0)
+
+    proj_3d = umap_3d.fit_transform(X, y=y)
+
+    fig = px.scatter_3d(proj_3d, x=0, y=1, z=2, color=y)
+
+    fig.update_layout(transition_duration=500, height=1000)
+    fig.update(layout_coloraxis_showscale=False)
+    fig.update_traces(marker_size=2)
+
+    return fig
+```
+
+### Run the App
+
+Finally, define the code to run the app. The important points here are the host and port numbers. For deployments to work on Saturn Cloud, the host value must be `"0.0.0.0"` and the port number `"8000"`. 
```

Comment:
It might be worth changing the wording to point out the `0.0.0.0` isn't a Saturn Cloud requirement, no computer would let traffic come in if you don't set that host